packages/core/lib/test/async-every.test.js

```diff
@@ -3,53 +3,61 @@ const { describe, it, expect } = global
 const every = require('../async-every')
 
 describe('async-every', () => {
-  it('handles iterator errors', done => {
-    every([1], (item, cb) => cb(new Error('derp')), (err, result) => {
-      expect(err)
-      expect(err.message).toBe('derp')
-      done()
+  it('handles iterator errors', () => {
+    return new Promise(resolve => {
```

Comment:
the lint rules prefer returning a promise over the done callback approach - see https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-test-callback.md

Reply:
Can we use `async`/`await` instead?

Reply:
That doesn't look to be supported by this eslint rule. I'm going to disable this rule for now and we can come back to it later.